tests/resources/Harbor-Pages/Project-Members.robot

```diff
@@ -22,14 +22,20 @@ ${HARBOR_VERSION}  v1.1.1
 *** Keywords ***
 Go Into Project
     [Arguments]  ${project}  ${has_image}=${true}
-    Wait Until Element Is Visible  ${search_input}
+    Wait Until Element Is Visible And Enabled  ${search_input}
     Input Text  ${search_input}  ${project}
     Wait Until Page Contains  ${project}
-    Wait Until Element Is Visible  xpath=//*[@id='project-results']//clr-dg-cell[contains(.,'${project}')]/a
+    Wait Until Element Is Visible And Enabled  xpath=//*[@id='project-results']//clr-dg-cell[contains(.,'${project}')]/a
     Click Element  xpath=//*[@id='project-results']//clr-dg-cell[contains(.,'${project}')]/a
-    Run Keyword If  ${has_image}==${false}  Wait Until Element Is Visible  xpath=//clr-dg-placeholder[contains(.,\"We couldn\'t find any repositories!\")]
-    ...  ELSE  Wait Until Element Is Visible  xpath=//clr-dg-cell[contains(.,'${project}/')]
-    Capture Page Screenshot  gointo_${project}.png
+
+    :For  ${n}  IN RANGE  1  5
```

Comment:
Could you add some comments to explain why this is needed?

Reply:
OK, I will add some comments.